content/en/docs/tasks/administer-cluster/change-pv-access-mode-readwriteoncepod.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Change the Access Mode of a PersistentVolume to ReadWriteOncePod
+content_type: task
+weight: 90
```

Comment:
```suggestion
weight: 90
min-kubernetes-server-version: 1.22
```

Reply:
I think that should be fine. Just clarify that it is alpha in 1.22.

Reply:
Added an extra note to clarify. I feel I might be going overboard with these. @sftim what do you think?

Reply:
I see this is fixed.